srcpkgs/gimx/template

```diff
@@ -0,0 +1,80 @@
+# Template file for 'gimx'
+pkgname=gimx
+version=8.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gettext pkg-config"
+makedepends="libbluetooth-devel libcurl-devel libusb-devel libX11-devel libXi-devel
+ libxml2-devel mhash-devel ncurses-devel wxWidgets-gtk3-devel"
+depends="gimx-cli-${version}_${revision} gimx-firmware-${version}_${revision}
+ avrdude xdg-utils xterm"
+short_desc="Use a computer as a hub for your gaming devices"
+maintainer="amak <amak.git@outlook.com>"
+license="GPL-3.0-only"
+homepage="https://blog.gimx.fr/"
+distfiles="https://github.com/matlo/GIMX/releases/download/v${version}/gimx_${version}-1.tar.gz"
+checksum=783673435bd424ba49e6ad5911559a517f746e50dad0f10689f0dc53336a8e1e
+patch_args="-Np1"
+
+# silence unused function errors in loader/gimx-loader.cpp
+# that are triggered by archs not defined in info.h
+CPPFLAGS+=" -Wno-unused-function"
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" ncurses-devel"
+fi
+
+post_extract() {
+	# fix build with ncurses 6
+	vsed -i -e "s:ncursesw5-config:ncursesw6-config:" core/Makefile
+
+	# fix ncursesw/ncurses.h include
+	vsed -i -e "s:<ncursesw/ncurses.h>:<curses.h>:" core/display.c
+
+	# fix build with wxWidgets-gtk3
+	find . -iname Makefile -exec sed -i -e "s:wx-config:&-gtk3:" "{}" \;
```

Comment:
I'd ask upstream for an option in the makefile for this.

Reply:
I could, but I doubt any changes will be made. I asked the author about `ncursesw6-config` support and the response I got was that the "build system is archaic, it does not allow detecting library versions", and that my "only option is to manually edit the core/Makefile".